racingcar-java/src/test/java/step04/RacingCarTest.java

```diff
@@ -0,0 +1,61 @@
+package step04;
+
+import misson04.domain.RacingCar;
+import misson04.utils.RacingCarMovementStrategy;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingCarTest {
+
+    @Test
+    @DisplayName("자동차가 생성되었을 때, 기본 값이 1인지 테스트")
+    void initRacingCarTest() {
+        int DISTANCE_DEFAULT = 1;
+        RacingCar racingCar = new RacingCar();
+
+        assertThat(racingCar.getMovedDistance()).isEqualTo(DISTANCE_DEFAULT);
+    }
+
+    @DisplayName("자동차가 이미 이동한 거리를 주입했을 때, 주입한 거리랑 같은지 테스트")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3, 4, 5, 6, 7, 8, 9})
+    void moveRacingCarCorrectTest(int initDistance) {
+        RacingCar racingCar = new RacingCar(initDistance);
+
+        assertThat(racingCar.getMovedDistance()).isEqualTo(initDistance);
+    }
+
+    @Test
+    @DisplayName("자동차들이 올바르게 움직이는지 테스트")
+    void checkMovedRacingCarsCorrectTest() {
+        // given
+        int MOVED_CARS_DISTANCE = 2;
+        RacingCarMovementStrategy racingCarMovementStrategy = () -> true;
```

Comment:
```suggestion
        RacingCarMovementStrategy allwaysCarMovementStrategy = () -> true;
```
테스트라도 의미가 조금 명확하면 좋을 것 같네요